pokemongo_bot/stepper.py

```diff
@@ -21,42 +21,18 @@ def __init__(self, bot):
         self.config = bot.config
 
         self.pos = 1
-        self.x = 0
-        self.y = 0
-        self.dx = 0
-        self.dy = -1
         self.steplimit=self.config.maxsteps
         self.steplimit2 = self.steplimit**2
         self.origin_lat = self.bot.position[0]
         self.origin_lon = self.bot.position[1]
 
     def take_step(self):
-        position=(self.origin_lat,self.origin_lon,0.0)
-
+        position=(self.origin_lat, self.origin_lon, 0.0)
         self.api.set_position(*position)
-
-        for step in range(self.steplimit2):
-            #starting at 0 index
-            print('[#] Scanning area for objects ({} / {})'.format((step+1), self.steplimit**2))
-            if self.config.debug:
-                print('steplimit: {} x: {} y: {} pos: {} dx: {} dy {}'.format(self.steplimit2, self.x, self.y, self.pos, self.dx, self.dy))
-            # Scan location math
-            if -self.steplimit2 / 2 < self.x <= self.steplimit2 / 2 and -self.steplimit2 / 2 < self.y <= self.steplimit2 / 2:
-                position = (self.x * 0.0025 + self.origin_lat, self.y * 0.0025 + self.origin_lon, 0)
-                if self.config.walk > 0:
-                    self._walk_to(self.config.walk, *position)
-                else:
-                    self.api.set_position(*position)
-                print('[#] {}'.format(position))
-                with open('web/location.json', 'w') as outfile:
-                    json.dump({'lat': position[0], 'lng': position[1]}, outfile)
```

Comment:
Please, don't remover the location dump. It's used by our map. Put this under the `self.api.set_position` call at line 31